index.rst

```diff
@@ -21,10 +21,11 @@ Contents
 
 .. toctree::
    :maxdepth: 1
-   :caption: Distribution
-   :name: sec-distribution
-   
-   Distribution
+   :caption: Standards
+   :name: sec-standards
+
```

Comment:
I'm confused by this "standards" terminology. 

When I hear "standard" I get the impression we are describing how _somebody else_ should implement a standards compliant pre-rootfs environment. I don't think this is needed. I think there should only be THE halium pre-rootfs env. (Also I think the conclusion of the ongoing hybris-boot, halium-boot conversation should be that we agree on ONE, not that we eventually have multiple, standards-compliant ones, right?!)

I think it would be IMMENSELY helpful to have a written specification for what THE halium pre-rootfs env does. I think that would be the ideal basis to align the needs for the distributions and it could be the basis how to decide on the halium-boot conversation. But right now I'm not clear whether THIS document is that specification or whether it is something else.

Reply:
I agree that "One True Way" is likely the best way to go. However, there are some situations I'd like to remedy by using a standard rather than just documenting what halium-boot does.

Mostly, I want to have a world where we can guarantee compatibility between diverse pieces of software. I want the PRE to be a drop-in replacement (possibly with some tweaking to the image format).

Different distributions have different preferences on the specific software they'd like to use in their initramfs. One distribution may be *really* set on dracut for some reason. The standard is our way to be able to say, "Fine, use whatever you want, but make it compatible to this. Later, if you want to use what everyone else is using, it'll be drop-in".

Selfishly, I have some ideas floating around in my head that require maintaining compatibility with halium-boot while replacing it. Having everyone agree here means that I can try those without making everyone move *again*.